develop/toolkit/api-libraries/sidecar.md

```diff
@@ -1,4 +1,100 @@
 ---
 title: Sidecar Rest API
-description: TODO
----
\ No newline at end of file
+description: Learn about Substrate API Sidecar, a REST service that provides endpoints for interacting with Polkadot SDK-based chains and simplifies blockchain interactions.
+---
+
+# Sidecar API
+
+## Introduction
+
+The [Sidecar Rest API](https://github.com/paritytech/substrate-api-sidecar){target=\_blank} is a service that provides a REST interface for interacting with Polkadot SDK-based blockchains. With this API, developers can easily access a broad range of endpoints for nodes, accounts, transactions, parachains, and more.
+
+Sidecar functions as a caching layer between your application and a Polkadot SDK-based node, offering standardized REST endpoints that simplify interactions without needing complex, direct RPC calls. This approach is especially valuable for developers who prefer REST APIs or are building applications in languages with limited WebSocket support.
+
+Some of the key features of the Sidecar API include:
+
+- **REST API Interface** - provides a familiar REST API interface for interacting with Polkadot SDK-based chains
+- **Standardized Endpoints** - offers consistent endpoint formats across different chain implementations
+- **Caching Layer** - acts as a caching layer to improve performance and reduce direct node requests
+- **Multiple Chain Support** - works with any Polkadot SDK-based chain, including Polkadot, Kusama, and custom chains
+
+## Installation
+
+To install Substrate API Sidecar, use one of the following commands:
+
+=== "npm"
+
+    ```bash
+    npm install -g @substrate/api-sidecar
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm install -g @substrate/api-sidecar
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn global add @substrate/api-sidecar
+    ```
+
+!!! note
+    Sidecar API requires Node.js version 18.14 LTS or higher. Verify your Node.js version:
+
+    ```bash
+    node --version
+    ```
+
+    If you need to install or update Node.js, visit the [official Node.js website](https://nodejs.org/){target=\_blank} to download and install the latest LTS version.
+
+You can confirm the installation by running:
+
+```bash
+substrate-api-sidecar --version
+```
+
```

Comment:
Add a reference to the official installation guide: https://github.com/paritytech/substrate-api-sidecar?tab=readme-ov-file#npm-package-installation-and-usage